docker/RATIONALE.md

```diff
@@ -1,21 +1,43 @@
 # zipkin-docker rationale
 
-## Why do we add HEALTHCHECK for test images?
-
+## Why do we add HEALTHCHECK?
 While most of our images are not production, we still add HEALTHCHECK for a better ad-hoc
 and automation experience. Many of our setups will not operate without a service
 dependency: having HEALTHCHECK present makes triage and scripting a bit easier.
 
-HEALTHCHECK on our test image serves primarily two purposes:
+HEALTHCHECK on our test image serves primarily three purposes:
  * ad-hoc or scripted status of health using docker ps instead of knowing Kafka commands
  * to allow manual usage of the docker-compose v2 service_healthy condition
+ * support Docker Hub automated test service
 
 Ex. The following command can be used ad-hoc or in scripts without the user knowing Kafka:
 ```bash
 $ docker inspect --format='{{json .State.Health.Status}}' kafka-zookeeper
 "unhealthy"
 ```
 
+### Why do we change default timeouts?
+We changed timeouts in order to mark success faster.
+
+By default, the Docker health check runs after 30s, and if a failure occurs,
+it waits 30s to try again. This implies a minimum of 30s before the server is
+marked healthy.
+
+https://docs.docker.com/engine/reference/builder/#healthcheck
+
+We expect the server startup to take less than 10 seconds, even in a fresh
```

Comment:
I think this depends on your resources but still valuable info so I am not sure on how to improve it.